Makefile

```diff
@@ -577,6 +577,14 @@ update-gh-actions-go-version: ## Update Go version in GitHub action config.
 	$(QUIET) for fl in $(shell find .github/workflows -name "*.yaml" -print) ; do sed -i 's/go-version: .*/go-version: $(GO_IMAGE_VERSION)/g' $$fl ; done
 	@echo "Updated go version in GitHub Actions to $(GO_IMAGE_VERSION)"
 
+update-main-go-version: ## Update Go version in main.go.
```

Comment:
Instead of this being a hardcoded list, would a command like this work better ?

 ```
 find .  -name main.go -not -path "./vendor/*"   
./bugtool/main.go
./cilium-health/responder/main.go
./cilium-health/main.go
./cilium/main.go
./clustermesh-apiserver/main.go
./daemon/main.go
./hubble-relay/main.go
./operator/main.go
./plugins/cilium-docker/main.go
./tools/alignchecker/main.go
./tools/dev-doctor/main.go
./tools/maptool/main.go
./tools/mount/main.go
./tools/spdxconv/main.go
```

Reply:
Great suggestion, done.